src/index.html

```diff
@@ -15,31 +15,31 @@ <h1 class="font-semibold text-4xl">Leaderboard App</h1>
           </div>
           <div class="flex justify-between align-items-center mt-12 mb-4">
             <h2 class="font-semibold text-2xl">Recent Scores</h2>
-            <button type="button" class="border py-2 px-12">Refresh</button>
+            <button type="button" class="border py-2 px-12" id="refresh">
+              Refresh
+            </button>
           </div>
           <div class="w-full border p-6">
-            <ul>
-              <li class="my-2 font-semibold">Amadu Kamara: 100</li>
-              <li class="my-2 font-semibold">Samuel Amkam: 98</li>
-              <li class="my-2 font-semibold">Web Avenger 97</li>
-            </ul>
+            <ul class="leaderboard-list"></ul>
           </div>
         </div>
         <div class="mt-20 mx-10 w-4/12">
           <h2 class="font-semibold text-2xl">Add your score</h2>
-          <form class="mt-4">
+          <form class="mt-4" id="leaderborad-form">
             <div class="mb-3">
               <input
                 type="text"
                 placeholder="Your name"
-                class="border py-2 px-4 w-full"
+                class="border py-2 px-4 w-full rounded"
+                id="name"
               />
             </div>
             <div class="mb-3">
               <input
                 type="text"
                 placeholder="Your score"
-                class="border py-2 px-4 w-full"
+                class="border py-2 px-4 w-full rounded"
+                id="score"
```

Comment:
[OPTIONAL] I think you could validate that the form is being filled appropriately. This would bring a better user experience and ensure that the data we received is the expected one.

Reply:
Oh, yeah sure. 👍